### PR TITLE
indexer easy: add tx command count for CPS

### PR DESCRIPTION
**Pull request by gegaowp**
## Description 

Add command count to tx table on indexer to unblock CPS metric on Explorer.

## Test Plan 

CI

### CODE REVIEW

**Review by wlmyng — APPROVED** [review]
:ok:

File: crates/sui-indexer/src/models/transactions.rs [hunk 1]
```diff
@@ -80,7 +81,7 @@ impl TryFrom<SuiTransactionFullResponse> for Transaction {
     type Error = IndexerError;
 
     fn try_from(tx_resp: SuiTransactionFullResponse) -> Result<Self, Self::Error> {
-        let txn_json = serde_json::to_string(&tx_resp.transaction).map_err(|err| {
+        let tx_json = serde_json::to_string(&tx_resp.transaction).map_err(|err| {
```
Comment (wlmyng) [inline crates/sui-indexer/src/models/transactions.rs:84]: so `tx` going forward instead of `txn`?

Comment (gegaowp) [reply crates/sui-indexer/src/models/transactions.rs:84]: found out more tx in the codes, I was more used to txn though, so I will try to use tx to be consistent LOL

Comment (wlmyng) [reply crates/sui-indexer/src/models/transactions.rs:84]: ah i see, tbh i prefer `txn` too, maybe we need 2f+1 to determine :P

**Pull request merged**